src/index.ts

```diff
@@ -28,19 +32,17 @@ import {
 } from "coc.nvim";
 import { spawn, ChildProcessPromise } from "promisify-child-process";
 import {
-  ExitNotification,
   ExecuteCommandRequest,
   Location,
-  Range,
-  ShutdownRequest
+  Range
 } from "vscode-languageserver-protocol";
 import * as path from "path";
 import { MetalsFeatures } from "./MetalsFeatures";
 import DecorationProvider from "./decoration";
 import { InputBoxOptions } from "./portedProtocol";
 
 export async function activate(context: ExtensionContext) {
-  detechLauncConfigurationChanges();
```

Comment:
`Launc` 😥

Reply:
detech 😄

Reply:
Haha, I didn't even notice that until now. I need to go back to school and learn to spell.